src/hmrc-content-guide/forms-detailed-guide/index.njk

```diff
@@ -5,56 +5,99 @@ layout: twoColumnPage.njk
+detailed guide, use either an inline link or a <a href="https://design.tax.service.gov.uk/hmrc-content-guide/green-button/" class="govuk-link">green button</a>.
+</p>
+
+<p class="govuk-body">
+These forms will not require an accessibility statement.
+</p>
+
+<h3 class="govuk-heading-m">Open Document Formats</h3>
+
+<p class="govuk-body">
+If the form is in Open Document Format it will not require an accessibility statement.
+</p>
+
+<p class="govuk-body">
+You must [use the publication box !@n markdown in
+Whitehall](<a href="https://www.gov.uk/guidance/how-to-publish-on-gov-
+uk/markdown#publication-box" class="govuk-link">https://www.gov.uk/guidance/how-to-publish-on-gov-
+uk/markdown#publication-box</a>).
+</p>
+
+<h3 class="govuk-heading-m">Print and post forms</h3>
+
+<p class="govuk-body">
+These will be hosted on <a href="https://public-online.hmrc.gov.uk/lc/content/" class="govuk-link">https://public-online.hmrc.gov.uk/lc/content/</a>.
+</p>
+
+<p class="govuk-body">
+If the form is a print and form, include the following markdown:
+</p>
+
+{{
     codeSnippet({
-      code: 'Read the [accessibility statement for HMRC forms](https://www.gov.uk/guidance/accessibility-statement-for-hmrc-interactive-forms).',
+      code: 's1. You need to open and complete this form online. As you cannot save your progress, you may want to get all your information together before you start.
+
+s2. Fill in [name of form](LINK).
+
+s3. Print and post it to HMRC, using the postal address shown on the form.
+
+^This file may not be suitable if you use assistive technology – such as a screen reader. If you need a more accessible format email <different.format@hmrc.gov.uk> and tell us what format you need. It will help if you tell us what assistive technology you use. Read the [accessibility statement for HMRC forms](https://www.gov.uk/guidance/accessibility-statement-for-hmrc-interactive-forms).',
       canCopy: true
     })
   }}
 
-<p class="govuk-body">
-If the form is not translated into Welsh, always include:
+  <p class="govuk-body">
+You should check with the DFOM (Digital Forms and Outbound Mail) team if you
+need to include the following:
 </p>
 
   {{
     codeSnippet({
-      code: 'Email HMRC to [ask for the form in Welsh (Cymraeg)](https://www.gov.uk/government/organisations/hm-revenue-customs/contact/welsh-language-helplines#email).',
+      code: 'If the form does not open, then [contact the Online services helpdesks for more help](https://www.gov.uk/government/organisations/hm-revenue-customs/contact/online-services-helpdesk).',
       canCopy: true
     })
   }}
 
-<h3 class="govuk-heading-m">Forms that are attachments</h3>
+  <h3 class="govuk-heading-m">PDF forms that can be completed on-screen</h3>
 
-<p class="govuk-body">
-If you’re going to attach a file, you must use the publication box !@n markdown.
+  <p class="govuk-body">
+  You must [use the publication box !@n markdown in
```

Comment:
Think simplify to...

You must use the [publication box]([https://www.gov.uk/guidance/how-to-publish-on-gov- uk/markdown#publication-box) !@n markdown in Whitehall to attach the PDF form.